nights/2024-04-12-README.md

```diff
@@ -1,26 +1,66 @@
 # 2024-04-12
 
-## Reviewed by:   your_name_here
+## Reviewed by:   Jake Mailhot
 
 ## Raw image gallery: http://physics.mnstate.edu/feder_gallery/2024-04-12
 
 ## Unusual images?
 
 Are there any images that look unusual? List the file name of any unusual images for this night here, with description:
 
-+ `crazy-object-001R.fit` -- not sure what this is an image of
-+ `another-crazy-object-003B.fit` -- this looks more like a flat than a science image.
-+ `full-moon-005B.fit` -- nice satellite or plane track in here.
+TIC_81247877-S001-R001-C001-rp.fit - slight drag on the image
+TIC_81247877-S001-R001-C007-rp.fit - Unclear
+TIC_81247877-S001-R004-C007-rp.fit - Uneven clarity
+TIC_81247877-S001-R004-C008-rp.fit - Unclear
+TIC_81247877-S001-R004-C009-rp.fit - Unclear
+TIC_81247877-S001-R005-C002-rp.fit - Unclear
+TIC_81247877-S001-R005-C003-rp.fit - Unclear
+TIC_81247877-S001-R005-C004-rp.fit - Unclear
+TIC_81247877-S001-R005-C005-rp.fit - Unclear
+TIC_81247877-S001-R005-C006-rp.fit - Unclear
+TIC_81247877-S001-R008-C001-rp.fit - Satallite
+TIC_81247877-S001-R008-C003-rp.fit - Satallite
+ey-uma-S001-R001-C001-B.fit - Looks like a flat
+ey-uma-S001-R001-C001-V.fit - Looks like a flat
+ey-uma-S001-R001-C001-rp.fit - Looks like a flat
+ey-uma-S001-R001-C002-rp.fit - Looks like a flat
+ey-uma-S001-R001-C003-rp.fit - Looks like a flat
+ey-uma-S001-R001-C004-rp.fit - Looks like a flat
+ey-uma-S001-R001-C005-rp.fit - Looks like a flat
+ey-uma-S001-R001-C006-rp.fit - Looks like a flat
+ey-uma-S001-R001-C007-rp.fit - Looks like a flat
+ey-uma-S001-R001-C007-rp.fit - Looks like a flat
+ey-uma-S001-R001-C008-rp.fit - Looks like a flat
+ey-uma-S001-R001-C009-rp.fit - Looks like a flat
+ey-uma-S001-R001-C010-rp.fit - Looks like a flat
+ey-uma-S001-R001-C011-rp.fit - Grainy
```

Comment:
these images look ok- maybe a little out of focus - but our images do look a little grainy all the time